sig-storage/README.md

```diff
@@ -84,9 +84,6 @@ The following [subprojects][subproject-definition] are owned by sig-storage:
 ### kubernetes-cosi
 - **Owners:**
   - [kubernetes-sigs/container-object-storage-interface-api](https://github.com/kubernetes-sigs/container-object-storage-interface-api/blob/master/OWNERS)
```

Comment:
@Priyankasaggu11929 @xing-yang the API repo is also being deprecated, and this repo is replacing all 4 of them. Would this be an appropriate change? Or am I misunderstanding a part of the process?

```suggestion
  - [kubernetes-sigs/container-object-storage-interface](https://github.com/kubernetes-sigs/container-object-storage-interface/blob/master/OWNERS)
```

Reply:
That is my "suggested change". In the diff, the API is still remaining